test/localPRG_test.cpp

```diff
@@ -1316,6 +1316,303 @@ TEST(LocalPRGTest, find_alt_path) {
 
 }
 
+// TODO: refactor some tests here to make use of mocks
```

Comment:
Rather than have the TODO here, be more specific about which tests need refactoring and tag them with the TODO

Reply:
I should have removed this TODO, as unit tests might be refactored to be compliant with the conventions, if we accept to follow them. So all unit tests will be refactored, even if the majority will be minor refactoring (e.g. declaring common setups in Fixtures, maybe using mock, and splitting a complex unit test into several simple ones).